lib/sensu-handler.rb

```diff
@@ -122,7 +122,7 @@ def api_settings
       @api_settings
     end
 
-    def api_request(method, path)
+    def api_request(method, path, &_blk)
```

Comment:
Honestly since I would probably never write something like this I am not sure but I think it should be `_&blk`? maybe someone who is more of a ruby guy than me knows @eheydrick ?